web/angular-wallet/src/app/layout/components/i18n/locales.ts

```diff
@@ -0,0 +1,92 @@
+
+import { registerLocaleData } from '@angular/common';
+import localeBg from '@angular/common/locales/bg';
+import localeBgExtra from '@angular/common/locales/extra/bg';
+import localeCa from '@angular/common/locales/ca';
+import localeCaExtra from '@angular/common/locales/extra/ca';
+import localeCs from '@angular/common/locales/cs';
+import localeCsExtra from '@angular/common/locales/extra/cs';
+import localeDe from '@angular/common/locales/de';
+import localeDeExtra from '@angular/common/locales/extra/de';
+import localeEl from '@angular/common/locales/el';
+import localeElExtra from '@angular/common/locales/extra/el';
+import localeEs from '@angular/common/locales/es';
+import localeEsExtra from '@angular/common/locales/extra/es';
+import localeFi from '@angular/common/locales/fi';
+import localeFiExtra from '@angular/common/locales/extra/fi';
+import localeFr from '@angular/common/locales/fr';
+import localeFrExtra from '@angular/common/locales/extra/fr';
+import localeGl from '@angular/common/locales/gl';
+import localeGlExtra from '@angular/common/locales/extra/gl';
+import localeHi from '@angular/common/locales/hi';
+import localeHiExtra from '@angular/common/locales/extra/hi';
+import localeHr from '@angular/common/locales/hr';
+import localeHrExtra from '@angular/common/locales/extra/hr';
+import localeId from '@angular/common/locales/id';
+import localeIdExtra from '@angular/common/locales/extra/id';
+import localeIt from '@angular/common/locales/it';
+import localeItExtra from '@angular/common/locales/extra/it';
+import localeJa from '@angular/common/locales/ja';
+import localeJaExtra from '@angular/common/locales/extra/ja';
+import localeLt from '@angular/common/locales/lt';
+import localeLtExtra from '@angular/common/locales/extra/lt';
+import localeNl from '@angular/common/locales/nl';
+import localeNlExtra from '@angular/common/locales/extra/nl';
+import localePl from '@angular/common/locales/pl';
+import localePlExtra from '@angular/common/locales/extra/pl';
+import localePtBr from '@angular/common/locales/pt-AO';
+import localePtBrExtra from '@angular/common/locales/extra/pt-AO';
```

Comment:
This is probably not right, but `pt-BR` is not an option. Will ask about this at ng-conf in 2 weeks.